CreatubblesAPIClient/Sources/Core/Data/NewCreationData.swift

```diff
@@ -52,6 +53,7 @@ open class NewCreationData: NSObject {
     open var creatorIds: Array<String>?
     open var creationYear: Int?
     open var creationMonth: Int?
+    open var asset: AVURLAsset?
```

Comment:
The current logic depends too much on `NewCreationData` class. We should prepare an alternate object responsible for passing this kind of information, but as you said it won't be that simple.